src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -45,9 +45,10 @@
           inputFileLabel="Continuation authorization"
           :maxSize="MAX_FILE_SIZE"
           :pdfPageSize="PdfPageSize.LETTER_SIZE"
+          :hint="authorization.files[index]?.file ? 'File uploaded.' : undefined"
           :inputFile="authorization.files[index]?.file"
           :showErrors="getShowErrors"
-          :customErrorMessage="fileUploadCustomErrorMsg"
+          :customErrorMessage.sync="customErrorMessage[index]"
```

Comment:
Two changes here:
- allows for sync from child back to parent (doesn't have to be used, which retains compatibility for other filings that use FileUploadPreview)
- supports different error messages for every instance of this component (up to 5)

Reply:
Yup, and eventually we're going to have to move away from this as it's deprecated in Vue 3 if I recall. A problem for another day.

Reply:
I think we will be able to use a v-model for this in Vue3.